spec/features/navigating_browse_pages_spec.rb

```diff
@@ -20,15 +20,15 @@
   end
 
   def given_there_are_browse_pages
-    create(:mainstream_browse_page, :published, title: "Money and Tax")
-    citizenship = create(:mainstream_browse_page, :published, title: "Citizenship")
-    create(:mainstream_browse_page, parent: citizenship, title: "Voting")
-    british_citizenship = create(:mainstream_browse_page, parent: citizenship, title: "British citizenship")
+    @money_and_tax = create(:mainstream_browse_page, :published, title: "Money and Tax")
```

Comment:
is there a reason why these needed to be assigned?

Reply:
It's used on line 44 and 45 to check it's rendered on the show page